capabilities.tf

```diff
@@ -1,69 +1,70 @@
-// This file is replaced by code-generation using 'capabilities.tf.tmpl'
```

Comment:
Revert this file. This is auto-generated.

Reply:
Wups, thanks for catching this